students/westobrien/Lesson03/list_lab.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+
+list1 = ["Apples", "Pears", "Oranges", "Peaches"]
+print(list1)
+response = input("Please type in another fruit: ")
+list1.append(response)
+print(list1)
+number1 = input("please type a number (1-5): ")
```

Comment:
len() is a good idea here to show the length of the list instead of hard coding 1-5